app/views/videos/show.html.haml

```diff
@@ -10,4 +10,33 @@
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]])
```

Comment:
The solution video shows a way to do this with pluralize `= f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]}), { label: "Rate this video" }` Plus you can use the inline label.